internal/logic_handlers/organizationlogic/adapters.go

```diff
@@ -0,0 +1,20 @@
+package organizationlogic
+
+import (
+	"context"
+	"github.com/dembygenesis/local.tools/internal/model"
+	"github.com/dembygenesis/local.tools/internal/persistence"
+)
+
+//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 -generate
+
+//counterfeiter:generate . persistor
+type persistor interface {
+	GetOrganizations(ctx context.Context, tx persistence.TransactionHandler, filters *model.OrganizationFilters) (*model.PaginatedOrganizations, error)
+	CreateOrganization(ctx context.Context, tx persistence.TransactionHandler, organization *model.Organization) (*model.Organization, error)
+	AddOrganization(ctx context.Context, tx persistence.TransactionHandler, organization *model.CreateOrganization) (*model.Organization, error)
```

Comment:
Duplicate, you guys can decide whether create or add should be preserved.

Reply:
Alright. We've decided to remove the CreateOrganization.